docs/source/io_formats/statepoint.rst

```diff
@@ -92,6 +92,26 @@ The current version of the statepoint file format is 16.0.
            - **width** (*double[]*) -- Width of each mesh cell in each
              dimension.
 
+**/tallies/filters/**
```

Comment:
Can you update the statepoint version in src/constants.F90:19 and openmc/statepoint.py:12?